src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -55,13 +61,51 @@
     public static final EditCommand.EditPersonDescriptor DESC_AMY;
     public static final EditCommand.EditPersonDescriptor DESC_BOB;
 
+
+    // ---------------------- Session Testing -------------------------------
+    public static final String VALID_SESSIONNAME_REC = "recitation1";
+    public static final String VALID_SESSIONNAME_CON = "midterm consultation";
+    public static final String VALID_SESSIONDATE_REC = "12/7/2017";
+    public static final String VALID_SESSIONDATE_CON = "27/11/2015";
+
+    public static final String VALID_SESSIONNAME_REC_DESC = " " + PREFIX_SESSIONNAME + VALID_SESSIONNAME_REC;
+    public static final String VALID_SESSIONNAME_CON_DESC = " " + PREFIX_SESSIONNAME + VALID_SESSIONNAME_CON;
+    public static final String VALID_SESSIONDATE_REC_DESC = " " + PREFIX_SESSIONDATE + VALID_SESSIONDATE_REC;
+    public static final String VALID_SESSIONDATE_CON_DESC = " " + PREFIX_SESSIONDATE + VALID_SESSIONDATE_CON;
```

Comment:
Would it make more sense if all instances of `SESSIONNAME` got changed to `SESSION_NAME`? (same for `SESSIONDATE` and `SESSION_DATE`?